utils.py

```diff
@@ -20,3 +20,22 @@ def log(self, string):
         if self._enabled:
             prefix = "[terminal_view debug] [%.3f] " % (time.time())
             print(prefix + string)
+
+
+class TerminalViewManager():
```

Comment:
Not a fan of Manager classes 😕 could you try going the ViewEventListener route instead?